R/repo-helper.R

```diff
@@ -68,6 +68,9 @@ create_demo_project <- function(open = rlang::is_interactive(),
         rlang::with_interactive(
           {
             usethis::use_news_md()
+            news_lines <- readLines("NEWS.md")
```

Comment:
the changes meant we now have to create a demo project with a preambled NEWS.md for tests to pass :-)

Reply:
For the future, could this also be a comment in the code?

Reply:
which code?

Reply:
aaah I should do as I say then https://blog.r-hub.io/2023/01/26/code-comments-self-explaining-code/#think-twice-before-adding-a-comment-on-your-own-pr :grin: :see_no_evil: :sweat_smile: